src/exponential.jl

```diff
@@ -75,7 +75,21 @@ An interval matrix that encloses ``B := αA + βA^2``.
 
 ### Algorithm
 
-This a variation of the algorithm in [1, Section 6].
+This a variation of the algorithm in [1, Section 6]. If ``A = (aᵢⱼ)`` and
+``B = := αA + βA^2 = (bᵢⱼ)``, the idea is to compute each ``bᵢⱼ`` by factoring
+out repeated expressions (thus the term *single-use expressions*).
+
+First, let ``i ≡ j``. In this case,
+
+```math
+bᵢᵢ = β\\sum_\\{k, k ≠ i} a_{ik} a_{ki} + (α + βa_{ii}) a_{ii}.
```

Comment:
I know it is totally equivalent, but the code is written in `j`, so maybe use the same symbol.